net/socket/connect.c

```diff
@@ -140,7 +140,7 @@ int psock_connect(FAR struct socket *psock, FAR const struct sockaddr *addr,
   if (ret >= 0)
     {
       FAR struct socket_conn_s *conn = psock->s_conn;
-      conn->s_flags |= _SF_CONNECTED;        
```

Comment:
Strange, checkpatch.sh didn't find this issue:
$ ./tools/checkpatch.sh -f net/rpmsg/rpmsg_sockif.c

There should be some BUG in the nxstyle.c

Reply:
nxstyle is hard to find the check duplication.